hangman.js

```diff
@@ -1 +1,21 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word = prompt("Enter a word.");
+console.log("word:" + word);
+var dashWord = "-";
+for (i = 1; i <word.length; i ++) {
+  dashWord += "-";
+}
+// var dashWord = "-----";
+alert("Word: " + dashWord + " Guess a letter");
+
+ var word = "pizza"
+ var gameOver = false;
+ var guess = 8;
+  while (gameOver == false){
+    var guess = prompt("Enter a Letter");
+    }
```

Comment:
you can end the while loop if there are no dashes in the output of `dashWord()`. e.g.

```js
if(!dashWord(word, guesses).match(/-/)){
  gameOver = true
}
```